pages/public_cloud/containers_orchestration/managed_kubernetes/configuring-multi-attach-persistent-volumes-with-ovh-nas-ha/guide.fr-fr.md

```diff
@@ -40,10 +64,19 @@ $ kubectl get nodes -o jsonpath='{ $.items[*].status.addresses[?(@.type=="Intern
 51.77.204.175 51.77.205.79
 ```
 
+#### Your cluster is installed with Private Network with a gateway
+
+Because your nodes are configured to be routed by the private network gateway, you need to allow the gateway IP address to the ACLs.
+By using Public Cloud Gateway through our Managed Kubernetes Service, Public IPs on nodes are only for management purposes : [MKS Known Limits](/pages/public_cloud/containers_orchestration/managed_kubernetes/known-limits)
+
+Get your Gateway's Public IP :
+
+Public Cloud > Select your tenant > Network / Gateway > Public IP
```

Comment:
No word about the following APIv6 endpoint? ` /cloud/project/{serviceName}/region/{regionName}/gateway/{id}`